.ci/scripts/setup-ios.sh

```diff
@@ -0,0 +1,33 @@
+#!/bin/bash
+# Copyright (c) Meta Platforms, Inc. and affiliates.
+# All rights reserved.
+#
+# This source code is licensed under the BSD-style license found in the
+# LICENSE file in the root directory of this source tree.
+
+set -exu
+
+# This script follows the instructions from GitHub to install an Apple certificate
+# https://docs.github.com/en/actions/use-cases-and-examples/deploying/installing-an-apple-certificate-on-macos-runners-for-xcode-development
+
+CERTIFICATE_PATH="${RUNNER_TEMP}"/build_certificate.p12
```

Comment:
How is the $RUNNER_TEMP set?

Reply:
That's one of the default env variable set by GitHub, usually it's a sub-directory inside runner installation dir https://docs.github.com/en/actions/writing-workflows/choosing-what-your-workflow-does/store-information-in-variables#default-environment-variables